wordpress-mobile/WordPress-Android
# Issue/14094 backup remove notify me

Fixes #14094 (BackupDownload) and #14088 (restore)

This PR removes the "Let me know when you are finished!" buttons from the progress view within the Backup Download and Restore processes.

|restore |backup download
|---|---|
|![restore-light](https://user-images.githubusercontent.com/506707/108277925-2e1a6d80-7148-11eb-9342-54d0380c4bf0.png)|![backup-download-light](https://user-images.githubusercontent.com/506707/108277927-2fe43100-7148-11eb-9c6b-d565dcbc8ff7.png)

|restore |backup download
|---|---|
|![restore-dark](https://user-images.githubusercontent.com/506707/108278009-5904c180-7148-11eb-8dfd-efff4c098d7e.png)|![backup-download-dark](https://user-images.githubusercontent.com/506707/108278012-5a35ee80-7148-11eb-9f58-80e62b8eda30.png)

To test:
Restore Process
- Launch the app
- Navigate to Activity Log
- Locate an activity log list item that can be restored
- Tap menu -> Restore to this point
- Tap Restore site
- Tap confirm
- Note
- - There is no longer a "Let me know when you are finished button"
- - The footnote is shown with the info icon to the left

------------------------------------
Backup Download
- Launch the app
- Navigate to Activity Log
- Locate an activity log list item that a downloadable backup file can be created
- Tap menu -> Download backup
- Tap Restore site
- Tap confirm
- Note
- - There is no longer a "Let me know when you are finished button"
- - The footnote is shown with the info icon to the left
------------------------------------

PR submission checklist:
